<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EngraveWidget</class>
 <widget class="QDockWidget" name="EngraveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>421</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Engraving</string>
  </property>
  <widget class="QWidget" name="dockWidgetContents">
   <layout class="QGridLayout" name="gridLayout" columnstretch="1">
    <item row="1" column="0">
     <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0,0,0">
      <item row="3" column="3">
       <widget class="QSlider" name="rot_slider">
        <property name="minimum">
         <number>-179</number>
        </property>
        <property name="maximum">
         <number>179</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="2" column="3">
       <widget class="QSlider" name="scal_slider">
        <property name="maximum">
         <number>300</number>
        </property>
        <property name="singleStep">
         <number>1</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
        <property name="sliderPosition">
         <number>100</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="t_up_pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Polyhedron_3.qrc">
          <normaloff>:/cgal/icons/resources/up.png</normaloff>:/cgal/icons/resources/up.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="t_left_pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Polyhedron_3.qrc">
          <normaloff>:/cgal/icons/resources/left_arrow.png</normaloff>:/cgal/icons/resources/left_arrow.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="3" column="2">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Rotation degree:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Scaling factor:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="t_down_pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Polyhedron_3.qrc">
          <normaloff>:/cgal/icons/resources/down.png</normaloff>:/cgal/icons/resources/down.png</iconset>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="t_right_pushButton">
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../../Polyhedron_3.qrc">
          <normaloff>:/cgal/icons/resources/right_arrow.png</normaloff>:/cgal/icons/resources/right_arrow.png</iconset>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QDoubleSpinBox" name="depth_spinBox">
        <property name="decimals">
         <number>6</number>
        </property>
        <property name="minimum">
         <double>0.000000000000000</double>
        </property>
        <property name="singleStep">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Engraving depth:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <widget class="QLineEdit" name="lineEdit">
      <property name="placeholderText">
       <string>An Example Text</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="visualizeButton">
          <property name="text">
           <string>Visualize</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="engraveButton">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Engrave</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <widget class="QGraphicsView" name="graphicsView"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../Polyhedron_3.qrc"/>
  <include location="../../Polyhedron_3.qrc"/>
  <include location="../../Polyhedron_3.qrc"/>
  <include location="../../Polyhedron_3.qrc"/>
 </resources>
 <connections/>
</ui>
